core/src/level/elements/tile/DoorTile.java

```diff
@@ -0,0 +1,68 @@
+package level.elements.tile;
+
+import basiselements.DungeonElement;
+import level.elements.ILevel;
+import level.tools.Coordinate;
+import level.tools.DesignLabel;
+import level.tools.LevelElement;
+
+public class DoorTile extends Tile {
+
+    public enum DoorColor{none, red, blue, yellow, green}
+    private DoorTile otherDoor;
+
+    /**
+     * Creates a new Tile.
+     *
+     * @param texturePath Path to the texture of the tile.
+     * @param globalPosition Position of the tile in the global system.
+     * @param designLabel Design of the Tile
+     * @param level The level this Tile belongs to
+     */
+    public DoorTile(
+            String texturePath, Coordinate globalPosition, DesignLabel designLabel, ILevel level) {
+        super(texturePath, globalPosition, designLabel, level);
+        this.levelElement = LevelElement.DOOR;
```

Comment:
`this` nur dann, wenn wirklich gebraucht. 

```suggestion
        levelElement = LevelElement.DOOR;
```